metric-collector/service-readiness/kubeflow-readiness.py

```diff
@@ -29,12 +29,9 @@ def metric_update(args, google_open_id_connect_token):
       headers={
           'Authorization': 'Bearer {}'.format(google_open_id_connect_token)
       })
-  if resp.status_code == 200:
-    KUBEFLOW_AVAILABILITY.set(1)
-    return 1
-  else:
-    KUBEFLOW_AVAILABILITY.set(0)
-    return 0
+  v = 1 if resp.status_code == 200 else 0
```

Comment:
nit: please put if and else on separate lines I think its more readable then this comprehension

Reply:
Thank You! I applied this!